src/extension/rn-extension.ts

```diff
@@ -1,6 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
+import * as fs from "fs";
+
+try {
+    fs.statSync(`${__filename}.map`); // We check if source maps are available
+    /* tslint:disable:no-var-requires */
+    require("source-map-support").install(); // If they are, we enable stack traces translation to typescript
+    /* tslint:enable:no-var-requires */
+} catch (exceptions) {
+    // If something goes wrong, we just ignore the errors
```

Comment:
Exactly, this is a debug-mode-only thing. In production the .map file won't be available, so the source-map-support line won't get executed. Even if for some miraculous reason the .map is there, we won't be shipping source-map-support in the .vsix because it's a dev dependency, so the require will stil fail, we'll eat the exception, and the user will be none the wiser